backend/app/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.1.5 on 2021-06-21 18:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Document',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('author', models.CharField(blank=True, max_length=255)),
+                ('date', models.IntegerField(blank=True, null=True)),
+                ('new_attributes', models.JSONField(blank=True, default=dict, null=True)),
+                ('text', models.TextField(blank=True)),
+                ('title', models.CharField(blank=True, max_length=255)),
+                ('word_count', models.PositiveIntegerField(blank=True, default=None, null=True)),
+                ('tokenized_text', models.JSONField(blank=True, default=None, null=True)),
+                ('word_counts_counter', models.JSONField(blank=True, default=dict, null=True)),
+                ('part_of_speech_tags', models.JSONField(blank=True, default=list, null=True)),
+            ],
+        ),
+    ]
```

Comment:
Thanks for making the migration! It's likely PR #5 will go into the main branch first, which has its own 0001_initial migration, so we'll probably need to delete this and remake it before merging.

Reply:
The 0001_inital.py file has been updated.

Reply:
Ah, that's been the pattern in the past, right? To update the existing migration file with new models? I'd like to move us toward the more conventional Django method of creating a new migration file for each sequential update. In that case, I think we should be able to reset this file back to what it is in the main branch (with just the `Pronoun` table created) then run `makemigrations` again to generate a `0002_` migration file. Would you mind trying that out?

Reply:
I've updated the migrations files. There's now a new `0002_document.py` file in the migrations folder.